backend/inspirehep/records/cli.py

```diff
@@ -114,9 +125,24 @@ def importer():
     type=click.File("rb"),
     help="Path to a JSON file, example: ``data/records/literature/999108.json``.",
 )
+@click.option(
```

Comment:
There should be 2 empty lines before this.

Reply:
why?

Reply:
PEP8. https://www.python.org/dev/peps/pep-0008/#blank-lines 2 blank lines should be between top level functions. Or do I interpret code wrong here?

Reply:
Ah sorry... My mistake. There are more click options. They were hidden here ;)